UPGRADE-6.3.md

```diff
@@ -214,20 +259,31 @@ Log-related menu items in the backend ("logs", "log channel definition") are now
 new sidebar menu, create menu items assigned to the corresponding tables. To display these items in the classic main
 menu (which itself is no longer visible by default), assign the tables to the "Logs" content box.
 
+## Sidebar Menu Items
+
+Sidebar menu items are (in contrast to the classic main menu) created independently from tables and backend modules.
+After migration some menu items that existed in the classic main menu might not yet be migrated to new sidebar menu
+items. Create menu items manually for these tables and modules. 
```

Comment:
Is there a way to find out if this is the case?

Reply:
This section wasn't completely correct, so I reworded.